hq/public/stylesheets/main.css

```diff
@@ -338,6 +338,30 @@ table.iam-report__table th {
   box-sizing: initial;
 }
 
+table.gcp-report__table td,
+table.gcp-report__table th {
+  min-height: 21px;
+  box-sizing: initial;
+  word-wrap: break-word;
+}
+
+table.gcp-report__table {
```

Comment:
Minor: I am not sure we need the `table` selector here. Security HQ currently tries to use [BEM](https://en.bem.info/methodology/quick-start/) (Block, Element, Modifier) to prevent the need for combined selectors.

You have a perfectly lovely BEM classname there, `gcp-report` is the block and `table` is the Element.